core/orchestrator.go

```diff
@@ -139,14 +139,11 @@ func (orch *orchestrator) ProcessPayment(payment net.Payment, manifestID Manifes
 			didReceiveErr = true
 		}
 
-		// TODO(yondonfu):
-		// Add ticket EV to the credit amount if any of the following conditions hold:
-		// - No error receiving the ticket
-		// - There is an error receiving the ticket, the type assertion on the error succeeds and the error is acceptable
-		// pmErr, ok := err.(pm.Error)
-		// if err == nil || (ok && pmErr.Acceptable()) {
-
-		// }
+		pmErr, ok := err.(pm.Error)
+		if err == nil || (ok && pmErr.Acceptable()) {
```

Comment:
Could we add tests for the following cases:

- When there is an error, but it is acceptable we should still credit B's balance
- When there is an error, but it is not acceptable we should NOT credit B's balance